46sshd/module-setup.sh

```diff
@@ -84,6 +84,12 @@ install() {
     echo systemd-tty-ask-password-agent >> "$initdir/root/.bash_history"
     chmod 600 "$initdir/root/.bash_history"
 
+    # sshd requires /var/log/lastlog for tracking login information
+    if [ ! -d "$initdir/var/log" ]; then
+        mkdir -p -m 0755 "$initdir/var/log"
+    fi
+    >> "$initdir/var/log/lastlog"
```

Comment:
Perhaps using `touch ...` is clearer?

Reply:
Gonna be honest with you, I had no idea what touch's real purpose was. I thought it created files. It's a much better solution to this.